src/core/payment/SubstituteAndExcludePayToAddressesForSuperCharge.ts

```diff
@@ -19,10 +19,12 @@ export class SubstituteAndExcludePayToAddressesForSuperCharge implements ISubsti
         //TODO: Find a different way to handle this without doing require in const
         const path = require('path');
 
+        const knownSubstitutePayToFile = path.join('src', 'data', 'knownSubstitutePayTo');
```

Comment:
mfSubstitutePayToFile might be a better name vs. known. 

I envision this file as well as the src/data/nps-address/Mina_foundation_Addresses being provided directly by MF on a public site so that the data in here can be validated by anyone running the code - they need something else to compare these files against (e.g. hosted on minafoundation.com or minaprotocol.com.)

Reply:
Done.